src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <!-- Big Search Bar -->
+      <form
+        class="search-bar search-bar--big"
+        data-qa="big"
+      >
+        <input
+          type="text"
+          class="search-bar__input search-bar__input--big"
+          placeholder="Try “Los Angeles“"
+          data-qa="keypress"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <!-- Small Search Bar -->
+      <form
+        class="search-bar search-bar--small"
+        data-qa="small"
+      >
+        <input
+          type="text"
+          class="search-bar__input search-bar__input--small"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
Consider adding a 'name' attribute to the input element for the small search bar to ensure that the form data can be properly submitted and processed.